tests/components/keymitt_ble/test_config_flow.py

```diff
@@ -173,8 +173,6 @@ async def test_no_link(hass: HomeAssistant) -> None:
         )
         await hass.async_block_till_done()
 
-    assert result3["type"] == FlowResultType.FORM
-    assert result3["step_id"] == "link"
-    assert result3["errors"] == {"base": "linking"}
+    assert result3["type"] == FlowResultType.CREATE_ENTRY
 
-    assert len(mock_setup_entry.mock_calls) == 0
+    assert len(mock_setup_entry.mock_calls) == 1
```

Comment:
And then you should restore the old behavior here:
```python
    assert result3["type"] == FlowResultType.FORM
    assert result3["step_id"] == "link"
    assert result3["errors"] == {"base": "linking"}
    assert len(mock_setup_entry.mock_calls) == 0
```

It doesn't make sense that this would (correctly) change in the first place, right?